src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterOptions.cs

```diff
@@ -29,7 +29,7 @@ public class PrometheusExporterOptions
         internal Func<DateTimeOffset> GetUtcNowDateTimeOffset = () => DateTimeOffset.UtcNow;
 
         private int scrapeResponseCacheDurationMilliseconds = 10 * 1000;
-        private IReadOnlyCollection<string> httpListenerPrefixes = new string[] { "http://*:80/" };
+        private IReadOnlyCollection<string> httpListenerPrefixes = new string[] { "http://0.0.0.0:9464/" };
```

Comment:
Interesting, it seems to be a spec bug to me.

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#prometheus-exporter

I believe `0.0.0.0` stands for "all IPv4 addresses" rather than "all addresses".
If "all addresses" is the desired behavior, `*` is the correct way.
I also doubt if "all addresses" is the right direction because it could be a security concern for some folks.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/issues/2275

Reply:
I agree on the potential security concern introduced by having all (IPv4) addresses as the default host. 
I guess we can wait a bit for the comments by folks regarding the above mentioning issue.

Reply:
See if this would get merged #2282.

Reply:
I think we've got a good number of approvals for the spec change PR here. Thank you.

Reply:
I think we've got a good number of approvals for the spec change PR here. Thank you.